packages/cdc-react/src/scss/_fonts.scss

```diff
@@ -0,0 +1,120 @@
+@use "typography" as *;
+
+@font-face {
+  font-family: "Open Sans";
+  src:
+    url("/fonts/opensans-light-webfont.woff2") format("woff2"),
+    url("/fonts/opensans-light-webfont.woff") format("woff");
+  font-weight: $font-weight-light;
+  font-style: normal;
+}
+
+//Light Italic
+@font-face {
+  font-family: "Open Sans";
+  src:
+    url("/fonts/opensans-lightitalic-webfont.woff2") format("woff2"),
+    url("/fonts/opensans-lightitalic-webfont.woff") format("woff");
+  font-weight: $font-weight-light;
+  font-style: italic;
+}
+
+//Regular
+@font-face {
+  font-family: "Open Sans";
+  src:
+    url("/fonts/opensans-regular-webfont.woff2") format("woff2"),
+    url("/fonts/opensans-regular-webfont.woff") format("woff");
+  font-weight: $font-weight-regular;
+  font-style: normal;
+}
+
+//Regular Italic
+@font-face {
+  font-family: "Open Sans";
+  src:
+    url("/fonts/opensans-italic-webfont.woff2") format("woff2"),
+    url("/fonts/opensans-italic-webfont.woff") format("woff");
+  font-weight: $font-weight-regular;
+  font-style: italic;
+}
+
+//Medium
+//TO DO: Generate the .woff file for this font
```

Comment:
What's the plan for the TODOs in this file?

Reply:
Yes, good point (I added these so I wouldn't forget, and here we are ;) ). I'll look into this now. If I end up going down a rabbit hole, maybe we can team up, or make another ticket.

Reply:
I worked some magic to convert the Medium and Medium Italic ttfs from Google Fonts to woff and woff2 formats, added them to the fonts folder, and removed the TO DOs.